packages/components/filters/filters-list/package.json

```diff
@@ -21,6 +21,12 @@
   "dependencies": {
     "@babel/runtime": "^7.20.13",
     "@babel/runtime-corejs3": "^7.20.13",
+    "@commercetools-uikit/design-system": "workspace:^",
+    "@commercetools-uikit/dropdown-menu": "workspace:^",
+    "@commercetools-uikit/filter-menu": "workspace:^",
+    "@commercetools-uikit/flat-button": "workspace:^",
+    "@commercetools-uikit/icons": "workspace:^",
+    "@commercetools-uikit/select-input": "workspace:^",
```

Comment:
pnpm uses this syntax, yarn should support it, but I did not encounter it yet here in ui-kit. (Some task seems to update the version numbers after a release.) If you're running into issues, use the latest published version number.